pkg/cli/commands/clusters/register.go

```
@@ -65,17 +88,29 @@ func ClusterRegister(client *paperspace.Client, createFilePath string) (string,
 				return "", err
 			}
 		}
-
-		if err := artifactsBucketPathPrompt.Run(); err != nil {
-			return "", err
-		}
 		if err := artifactsAccessKeyIDPrompt.Run(); err != nil {
 			return "", err
 		}
 		if err := artifactsSecretAccessKeyPrompt.Run(); err != nil {
 			return "", err
 		}
 
+		arn, err := getAWSSTSCallerArn(artifactsAccessKeyIDPrompt.Value,
+			artifactsSecretAccessKeyPrompt.Value,
+			region,
+		)
+
+		if err != nil {
+			println(fmt.Sprintf("Unable to validate AWS identity from credentails: %s", err))
```

You can return a custom error: errors.New("custom error")